Cargo.toml

```diff
@@ -78,6 +78,7 @@ single_use_lifetimes = "warn"
 [workspace.lints.clippy]
 unused_async = "deny"
 undocumented_unsafe_blocks = "warn"
+implicit_clone = "warn"
```

Comment:
The lint level is set to `warn`, should the PR reflect this more accurately instead of saying "it forbids"?

Reply:
This only means warn while doing development. Our CI forbids all warnings. So this automatically becomes an error in CI and a warning in local development.

Reply:
👍